src/reyna/Blackout/TimeRange.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Reyna
+{
+    public class TimeRange
+    {
+        public TimeRange(Time from, Time to)
+        {
+            this.From = from;
+            this.To = to;
+        }
+
+        public Time From { get; private set; }
+
+        public Time To { get; private set; }
+
+        public bool Contains(Time time)
+        {
+            if (this.To.AfterOrEqualTo(this.From))
+            {
+                return time.AfterOrEqualTo(this.From) && time.BeforeOrEqualTo(this.To);
+            }
+            else
+            {
+                return time.AfterOrEqualTo(this.From) || time.BeforeOrEqualTo(this.To);
```

Comment:
What is this about? Why can From be after To, what is that logic supposed to mean?